core-services/beneficiary-idgen/LOCALSETUP.md

```
@@ -0,0 +1,36 @@
+- [ ] Kafka
+  - [ ] Consumer
+  - [ ] Producer
+
+## Running Locally
+
+To run the IdGen services in your local system, you need to port forward below services
+
+```bash
+function kgpt(){kubectl get pods -n egov --selector=app=$1 --no-headers=true | head -n1 | awk '{print $1}'}
+kubectl port-forward -n egov $(kgpt egov-mdms-service) 8088:8080
+``` 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)